falkordb-sentinel/sentinel-entrypoint.sh

```
@@ -0,0 +1,223 @@
+    echo "Waiting for node host to resolve"
+    sleep 5
+  done
+}
+
+get_master() {
+  master_info=$(redis-cli -h $SENTINEL_HOST -p $SENTINEL_PORT -a $ADMIN_PASSWORD $TLS_CONNECTION_STRING --no-auth-warning SENTINEL get-master-addr-by-name $MASTER_NAME)
+
+  echo "Master Info: $master_info"
+
+  # If RUN_SENTINEL is 1 and could not connect to sentinel, wait and try again
+  if [[ $RUN_SENTINEL -eq 1 && -z $master_info]]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)